.github/workflows/ci_cd.yml

```diff
@@ -1,4 +1,4 @@
-name: Github Actions
+name: tests
```

Comment:
renamed so the badge on the README looks better :-)  I do want to create separate workflows for test, build, deploy anyway but I'll do that in a later PR